_posts/2023-01-18-11.0.0-release.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Apache Arrow 11.0.0 Release"
+date: "2023-01-18 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 11.0.0 release. This covers
+over 3 months of development work and includes [**423 resolved issues**][1]
+from [**95 distinct contributors**][2]. See the [Install Page](https://arrow.apache.org/install/)
+to learn how to get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 10.0.0 release, Ben Baumgold, Will Jones, Eric Patrick Hanson,
+Curtis Vogt, Yang Jiang, Jarrett Revels, Raúl Cumplido, Jacob Wujciak,
+Jie Wen and Brent Gardner have been invited to be committers.
+Kun Liu have joined the Project Management Committee (PMC).
+
+Thanks for your contributions and participation in the project!
+
+## Columnar Format Notes
+
+## Arrow Flight RPC notes
+
+## C++ notes
+
+## C# notes
+
+## Go notes
+
+## Java notes
```

Comment:
@lwhite1 can you help with the Java notes?

Reply:
> @lwhite1 can you help with the Java notes?

yes. I will do it over the weekend if I don't finish it today

Reply:
```suggestion
## Java notes
* Implement support for writing compressed files ([#15223](https://github.com/apache/arrow/pull/15223))
* Improve performance by short-circuiting null checks when comparing non null field types ([#15106](https://github.com/apache/arrow/pull/15106))
* Several enhancements to dictionary encoding ([#14891](https://github.com/apache/arrow/pull/14891), ([#14902](https://github.com/apache/arrow/pull/14902), ([#14874](https://github.com/apache/arrow/pull/14874))
* Extend Table to support additional vector types ([#14573](https://github.com/apache/arrow/pull/14573))
* Enhance and simplify handling of allocation management by integrating C Data into allocator hierarchy ([#14506](https://github.com/apache/arrow/pull/14506))  
* Make ComplexCopier agnostic of specific implementation of MapWriter ([#14557](https://github.com/apache/arrow/pull/14557))
* Distribute Apple M1 compatible JNI libraries via mavencentral ([#14472](https://github.com/apache/arrow/pull/14472))
* Extend Table copy functionality, and support returning copies of individual vectors ([#14389](https://github.com/apache/arrow/pull/14389))
```